Dockerfile

```diff
@@ -73,7 +73,7 @@ SHELL ["/bin/bash", "-c"]
 
 # python3-yaml needed to run juju actions, xmlsec1 needed if UseNewSAMLLibrary is set to false (the default)
 RUN apt-get -qy update && \
-    apt-get -qy dist-upgrade && \
+    apt-get -qy upgrade && \
```

Comment:
The same command is executed twice here

Reply:
I'm going to refactor the Dockerfile in another PR. I just did the minimum here.

Reply:
I removed the duplicate lines